_targets.R

```diff
@@ -795,12 +797,22 @@ list(
 
   tar_target(od_utility_combined, {
 
+    # Load input objects:
+    tar_load(od_shopping)
+    tar_load(od_visiting)
+    tar_load(od_leisure)
+    tar_load(commute_stats)
+    tar_load(parameters)
+
     od_utility_combined = rbind(od_shopping, od_visiting, od_leisure) |>
       dplyr::slice_max(n = parameters$max_to_route, order_by = all, with_ties = FALSE)
+    sum(od_utility_combined$bicycle) / sum(od_utility_combined$all)
 
     # Get % cycling for commuting per zone
-    pcycle_national = sum(commute_stats$comm_orig_bicycle, na.rm = TRUE) /
```

Comment:
Heads-up @juanfonsecaLS1 this was the cause of the issue, pcycle_national here is in fact the regional level so for Edinburgh cycling was supressed.

Reply:
No, we want the % cycling for utility trips to be higher in Edinburgh than places with lower than national % cycling.